lib/client.js

```diff
@@ -210,43 +210,47 @@ class Client extends Dispatcher {
     resume(this, true)
   }
 
-  get connected () {
-    return (
-      this[kSocket] &&
-      this[kSocket].connecting !== true &&
-      // Older versions of Node don't set secureConnecting to false.
-      (this[kSocket].authorized !== false ||
-       this[kSocket].authorizationError
-      ) &&
-      !this[kSocket].destroyed
-    )
-      ? 1
-      : 0
+  get destroyed () {
+    return this[kDestroyed]
   }
 
-  get pending () {
+  get closed () {
+    return this[kClosed]
+  }
+
+  get [kPending] () {
     return this[kQueue].length - this[kPendingIdx]
   }
 
-  get running () {
+  get [kRunning] () {
     return this[kPendingIdx] - this[kRunningIdx]
   }
 
-  get size () {
+  get [kSize] () {
     return this[kQueue].length - this[kRunningIdx]
   }
 
-  // TODO: Make private?
-  get busy () {
-    return this[kNeedDrain] > 1
-  }
-
-  get destroyed () {
-    return this[kDestroyed]
+  get [kConnected] () {
+    return (
+      this[kSocket] &&
+      this[kSocket].connecting !== true &&
+      // Older versions of Node don't set secureConnecting to false.
+      (this[kSocket].authorized !== false ||
+       this[kSocket].authorizationError
+      ) &&
+      !this[kSocket].destroyed
+    )
+      ? 1
+      : 0
   }
 
-  get closed () {
-    return this[kClosed]
+  get [kBusy] () {
+    const socket = this[kSocket]
+    return (
+      (socket && (socket[kReset] || socket[kWriting])) ||
+      (this[kSize] >= (this[kPipelining] || 1)) ||
+      this[kPending] > 0
+    )
```

Comment:
The reason why I was skeptical in "removing" them was because those properties are needed elsewhere.

Reply:
They are not needed in public api